apps/api/src/prisma/schema.prisma

```diff
@@ -518,3 +518,13 @@ model ChangeNotificationSocketMap {
 
   @@index([environmentId, socketId])
 }
+
+model UserEmailChange {
+  id            String    @id @default(cuid())
+  userId        String    
+  newEmail      String
+  otp           String
+  createdOn     DateTime  @default(now())
+
```

Comment:
You would also need to add an `expitesOn` field. Additionally, set up an index on that field like this: `@@index([expiresAt], name: "expiresAt")`

Reply:
Added